src/caret_analyze/record/records_service/response_time.py

```diff
@@ -485,12 +498,29 @@ def to_best_case_stacked_bar(self) -> RecordsInterface:
 
             Columns
             - {columns[0]}
-            - {columns[1]}
-            - {...}
-            - {columns[n-1]}
+            - {'response_time'}
 
         """
-        return self._records.to_range_records('best')
+        return self._timeseries.to_best_case_records()
+
+    def to_worst_with_external_latency_case_records(self) -> RecordsInterface:
+        """
+        Calculate the worst-with-external-latency case records data for response time.
+
+        The worst-with-external-latency case in response time includes message flow latencies
```

Comment:
This sentence feels unclear in its expression and is difficult to understand; please consider using a different way to convey this sentence.

Reply:
Fixed it in [d9d1ce7](https://github.com/tier4/CARET_analyze/pull/355/commits/d9d1ce7accdc922dfe6e445c6623d3c23746ec5d).